bindings/pyroot_experimental/PyROOT/test/CMakeLists.txt

```diff
@@ -8,3 +8,8 @@ ROOT_ADD_PYUNITTEST(pyroot_pyz_ttree_branch_attr ttree_branch_attr.py
 ROOT_ADD_PYUNITTEST(pyroot_pyz_ttree_iterable ttree_iterable.py
                     COPY_TO_BUILDDIR TreeHelper.h)
 
+# TFile pythonisations
+ROOT_ADD_PYUNITTEST(pyroot_pyz_tfile tfile.py)
+
+# TDirectory pythonisations
+ROOT_ADD_PYUNITTEST(pyroot_pyz_tdirectory tdirectory.py)
```

Comment:
Do we also need separate tests for `TDirectoryFile`?

Reply:
Not sure.